csharp_dotnetcore/18.Multi-Lingual-Bot/MultiLingualBot.csproj

```diff
@@ -0,0 +1,86 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <CodeAnalysisRuleSet>MultiLingualBot.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="Microsoft.Azure.CognitiveServices.ContentModerator" Version="0.12.1-preview" />
+    <PackageReference Include="Microsoft.Azure.CognitiveServices.Language" Version="1.0.1-preview" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.TemplateManager" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Schema" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Graph" Version="1.10.0" />
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="all" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta008">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Dialogs\Cancel\Resources\CancelStrings.Designer.cs">
+      <DesignTime>True</DesignTime>
```

Comment:
Can all of these be removed? I don't see any such directory, or correspoinding resx files (MainStrings.resx, SigninStrings.resx, etc).

Reply:
@cleemullins absolutely